source/squidplus-proxy/README.md

```diff
@@ -0,0 +1,53 @@
+# _squidplus-proxy_ Content Pack
+
+## Summary
+
+The _squidplus-proxy_ Content Pack provides both client artefacts that acquire then post Squid access logs to a Stroom instance and Stroom content
+artefacts that will normalise the Squid access logs into the Stroom [`event-logging-schema`](https://github.com/gchq/event-logging-schema) format.
+
+This package does not use the standard Squid acces log format, but a bespoke format called _squidplus_. This format records additional information such
+as port numbers, complete request and response headers, additional status information, data transfer sizes and next hop host information.
+
+Client deployment information can be found in the supplied README file.
```

Comment:
also spelling

+This package does not use the standard Squid acces log format, but a bespoke format called _squidplus_. This format records additional information such

should be "access"